tests/unit/core/utils/test_index.js

```diff
@@ -63,795 +64,810 @@ import {
 } from 'tests/unit/helpers';
 
 
-describe('addonHasVersionHistory', () => {
-  function createAddonWithType(type) {
-    return createInternalAddon({ ...fakeAddon, type });
-  }
-
-  it('requires an addon', () => {
-    expect(() => {
-      addonHasVersionHistory();
-    }).toThrow('addon is required');
-  });
+describe(__filename, () => {
```

Comment:
You are braver than me for fixing the indent 👀

Reply:
yes, sorry for that actually. it makes the diff less readable..

Reply:
The PR does not add anything in this file anymore since I moved the logic + test cases to the `errorHandler`.